main.go

```diff
@@ -36,13 +37,13 @@ func main() {
 		flag.PrintDefaults()
 		os.Exit(1)
 	}
-	validDateFinEffectif, err := DateFinEffectif(*dateFinEffectif)
+	validDateFinEffectif, err := time.Parse("2006-01-02", *dateFinEffectif)
```

Comment:
Note: la date "2006-01-02" est à respecter telle quelle,  pour signifier qu'on souhaite parser selon le format YYYY-MM-DD, cf https://stackoverflow.com/questions/40425699/go-time-parse-getting-month-out-of-range-error#comment79175294_40426078